apps/modernization-ui/src/pages/addPatient/AddPatient.tsx

```diff
@@ -245,11 +245,9 @@ const AddPatient = () => {
                             }}>
                             <div className="width-full text-bold flex-row display-flex flex-align-center flex-justify">
                                 New patient
-                                <div className="button-group">
-                                    <Button className="padding-x-3 add-patient-button" type={'submit'}>
-                                        Save changes
-                                    </Button>
-                                </div>
+                                <Button className="padding-x-3 add-patient-button" type={'submit'}>
```

Comment:
nit: can you remove `padding-x-3`?  I am not sure where it came from, it's not defined in our styles nor the USWDS styles.